inc/admin/options.php

```diff
@@ -53,6 +53,7 @@ function rocket_after_save_options( $oldvalue, $value ) {
 		'analytics_enabled'           => true,
 		'sucury_waf_cache_sync'       => true,
 		'sucury_waf_api_key'          => true,
+		'manual_preload'              => true,
```

Comment:
@crystinutzaa Were you able to confirm that `'manual_preload'` is the root cause for this problem?

Reply:
@hellofromtonya Yes, I confirmed the `manual_preload` is the root cause of this issue. Because is not excluded , then each time the option is changed it will clean WP Rocket cache.